README.md

```diff
@@ -241,6 +242,11 @@ The destroy command supports the following options passed as keyword arguments:
   `var_files` are provided, all var files will be passed to terraform.
 * `var_files`: an array of paths to terraform var files; if both `var_file` and
   `var_files` are provided, all var files will be passed to terraform.
+  * `module_depth`: the depth of modules to show in the output; defaults to
+  showing all modules.
+* `json`: whether or not the output from the command should be in json format;
+  defaults to `false`.
+
```

Comment:
This looks like a copy paste error as the destroy command doesn't take these options.

Reply:
whoops... sorry about that.